packages/manager/src/features/Kubernetes/KubernetesClusterDetail/NodePoolsDisplay/NodePool.tsx

```diff
@@ -85,170 +75,96 @@ export const NodePool = (props: Props) => {
       expanded={accordionExpanded}
       heading={
         <Box
-          sx={{
-            alignItems: 'center',
-            display: 'flex',
-            justifyContent: 'space-between',
-            p: 0,
-          }}
+          alignItems="center"
+          display="flex"
+          justifyContent="space-between"
+          pr={1}
         >
-          <Box display="flex">
-            <Typography variant="h2">{typeLabel}</Typography>
-            <Divider
-              orientation="vertical"
-              sx={(theme) => ({
-                height: 16,
-                margin: `4px ${theme.spacing(1)}`,
-              })}
-            />
-            <Typography variant="h2">
+          <Stack
+            alignItems="center"
+            direction="row"
+            divider={<Divider flexItem orientation="vertical" />}
+            spacing={{ sm: 1.5, xs: 1 }}
+          >
+            <Typography variant="h3">{typeLabel}</Typography>
+            <Typography variant="h3">
               {pluralize('Node', 'Nodes', count)}
             </Typography>
-          </Box>
-          <Hidden lgUp>
+          </Stack>
+          <Stack alignItems="center" direction="row" spacing={1}>
             {autoscaler.enabled && (
-              <Box position="absolute" right={(theme) => theme.spacing(10)}>
-                <Typography mx={1}>
-                  (Min {autoscaler.min} / Max {autoscaler.max})
-                </Typography>
-              </Box>
+              <Typography mx={1}>
+                <Hidden mdDown>Autoscaling </Hidden>(Min {autoscaler.min} / Max{' '}
```

Comment:
suggestion: We could even do the mobile viewport here. Tablet/md has space for the word.

```suggestion
                <Hidden smDown>Autoscaling </Hidden>(Min {autoscaler.min} / Max{' '}
```

https://github.com/user-attachments/assets/c03d8888-456f-47d8-8226-2a7fc9e4958a